validate/src/command.rs

```diff
@@ -465,17 +516,17 @@ mod tests {
     }
 
     #[test]
-    fn test_chat_input_name() {
-        assert!(chat_input_name("hello-command").is_ok()); // Latin language
-        assert!(chat_input_name("Hello").is_err()); // Latin language with uppercase
-        assert!(chat_input_name("hello!").is_err()); // Latin language with non-alphanumeric
+    fn test_name_characters() {
+        assert!(name_characters("hello-command").is_ok()); // Latin language
+        assert!(name_characters("Hello").is_err()); // Latin language with uppercase
+        assert!(name_characters("hello!").is_err()); // Latin language with non-alphanumeric
 
         assert!(chat_input_name("\u{437}\u{434}\u{440}\u{430}\u{441}\u{442}\u{438}").is_ok()); // Russian
-        assert!(chat_input_name("\u{417}\u{434}\u{440}\u{430}\u{441}\u{442}\u{438}").is_err()); // Russian with uppercase
-        assert!(chat_input_name("\u{437}\u{434}\u{440}\u{430}\u{441}\u{442}\u{438}!").is_err()); // Russian with non-alphanumeric
+        assert!(name_characters("\u{417}\u{434}\u{440}\u{430}\u{441}\u{442}\u{438}").is_err()); // Russian with uppercase
```

Comment:
if you'd like, we can use `#![allow(clippy::non_ascii_literal)]` here

Reply:
I allowed the lint on the whole tests module, it did not work for only one test.